### PR TITLE
Fix debouncing of full screen and image display

### PR DESCRIPTION
**Pull request by rjs-picturepark**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts [hunk 1]
```diff
@@ -9,8 +9,12 @@ import {
   Inject,
   Optional,
   Injector,
+  OnInit,
 } from '@angular/core';
 import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
+import { throttleTime } from 'rxjs/internal/operators/throttleTime';
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts:15]: import from 'rxjs/operators'

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts [hunk 2]
```diff
@@ -9,8 +9,12 @@ import {
...
+import { Subject } from 'rxjs/internal/Subject';
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts:16]: import from 'rxjs'

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/interfaces/content-image-peview.interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { IShareItem } from '../../../../content-details-dialog/fullscreen.service';
```
Comment (tc4rdpp) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/interfaces/content-image-peview.interfaces.ts:1]: Missing an 'r' in the filename - 'peview'

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts [hunk 4]
```diff
@@ -89,6 +99,25 @@ export class ContentImagePreviewComponent extends BaseComponent implements OnCha
     return this.content.contentType === ContentType.Audio;
   }
 
+  ngOnInit() {
+    this.displayFullscreen.pipe(throttleTime(1000, undefined, { leading: true })).subscribe((displayItems) => {
```
Comment (rjs-picturepark) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/components/content-image-preview/content-image-preview.component.ts:103]: Unsubscribe

**Pull request merged**